lib/rouge/lexers/javascript.rb

```diff
@@ -239,7 +239,7 @@ def self.analyze_text(text)
 
       state :object_key do
         mixin :whitespace
-        mixin :has_string
+        rule /"(\\\\|\\"|[^"])*"/, Name::Tag
```

Comment:
Do you think you could use the same regex from `:has_string` though?  Maybe store it in a variable?